### DISCUSSION TITLE
How to rotate map in the direction of movement?

### DISCUSSION THREAD

**Discussion by printmaps** [devemux86/cruiser#649] [Q&A] [answered]
I'm using Cruiser 5.0.0 for hiking. The prefered use case is to follow a track without any navigation. How is it possible to rotate the map in the direction of movement?

PS: Map rotation works for me in navigation mode.

**Comment by devemux86** [comment, marked as answer]
Map rotation works in 2 modes:
- Navigation
- Follow mode

Follow mode can be enabled with long press on the location button at bottom right.

Then the map orientation mode can be selected with long press on the compass:
- Movement direction
- Compass direction (enable compass sensor in map settings)
- North is up
